ddtrace/settings/peer_service.py

```diff
@@ -23,19 +21,18 @@ def __init__(self, set_defaults_enabled=None, peer_service_mapping=None):
     @property
     def set_defaults_enabled(self):
         if self._set_defaults_enabled is None:
-            env_enabled = asbool(os.getenv("DD_TRACE_PEER_SERVICE_DEFAULTS_ENABLED", default=False))
+            env_enabled = _get_config("DD_TRACE_PEER_SERVICE_DEFAULTS_ENABLED", False, asbool)
             self._set_defaults_enabled = SCHEMA_VERSION == "v1" or (SCHEMA_VERSION == "v0" and env_enabled)
 
         return self._set_defaults_enabled
 
     @property
     def peer_service_mapping(self):
         if self._peer_service_mapping is None:
-            self._unparsed_peer_service_mapping = os.getenv("DD_TRACE_PEER_SERVICE_MAPPING", default="")
+            self._unparsed_peer_service_mapping = _get_config("DD_TRACE_PEER_SERVICE_MAPPING", "")
             self._peer_service_mapping = parse_tags_str(self._unparsed_peer_service_mapping)
 
         return self._peer_service_mapping
 
 
 _ps_config = PeerServiceConfig()
```

Comment:
`PeerServiceConfig` does not use envier, `_report_telemetry(_ps_config)` was previously a no-op. With this change it started to raise an exception

Reply:
Just curious, what was the exception? If `PeerServiceConfig` isn't a subclass of `En` like for `DynamicInstrumentationConfig`, does that mean we aren't automatically reporting on these env vars (besides when we make the `_get_config()` calls)?

Reply:
Yup the `_report_telemetry(_ps_config)` essentially did nothing. Migrating `PeerServiceConfig` is marked as a TODO, we just need someone to prioritize this task :homer-simpson-hides-in-bush-meme: